resources/sts/4.10/sts_installer_permission_policy.json

```diff
@@ -182,7 +182,8 @@
                 "ec2:DescribeVpcEndpointServicePermissions",
                 "ec2:DescribeVpcEndpointServices",
                 "ec2:ModifyVpcEndpointServicePermissions",
-                "kms:DescribeKey"
+                "kms:DescribeKey",
+                "cloudwatch:GetMetricData"
```

Comment:
Whats the justification for this metric? Also does this need to be added to the 4.9 installer role as well?

Reply:
https://coreos.slack.com/archives/C03B98SL8L8/p1649345512136449
Getting vCPU's usage for preflight check:
https://issues.redhat.com/browse/SDA-5138

Reply:
@pvasant can answer your 4.8/4.9 question

Reply:
That slack link isn't working for me. Can you please verify it?

SDA-5791 has none of this information. Can you please update it to reflect the justification?

Reply:
done

Reply:
@zvikorn we need to add it to all the versions.